service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -262,7 +262,7 @@ EOF
 # Update the PATH for container JupyterLab
 if [[ -n "${INSTANCE_CONTAINER}" ]]; then
```

Comment:
Please restructure this block slightly to always emit a section for the Terra-specific NOTEBOOK_CONFIG customizations:

```
# Add a marker for the Terra-specific customizations
cat << EOF >> "${NOTEBOOK_CONFIG}"

### BEGIN: Terra-specific customizations ###

EOF

# Update the PATH for container JupyterLab
if [[ -n "${INSTANCE_CONTAINER}" ]]; then

  cat << EOF >> "${NOTEBOOK_CONFIG}"

import os

os.environ['PATH'] = "${USER_HOME_LOCAL_BIN}:" + os.environ['PATH']
EOF

fi
```